item-descriptions.lua

```diff
@@ -36,24 +36,23 @@ vanilla item in the game.
 
 If "raw/scripts/item-descriptions.lua" exists, it will entirely replace this one.
 Instead, mods should use "raw/scripts/more-item-descriptions.lua" to add content or replace
-descriptions on a case-by-case basis.  If an item description cannot be found in
+descriptions on a case-by-case basis. If an item description cannot be found in
 the latter script, view-item-info will fall back to the former.
 ]]
 
 -- see http://dwarffortresswiki.org/index.php/cv:Item_token
---luacheck: global
 descriptions = {
-    AMULET = {  "An item of jewellery worn around the neck for it's aesthetic value.",
+    AMULET = {  "An item of jewellery worn around the neck for its aesthetic value.",
                 "An amulet does not interfere with wearing other equipment."},
     ANIMALTRAP = {
                 "This tiny trap is used by your trappers to catch vermin. Some dwarves",
                 "like vermin as pets - if your cats don't get them first. May be built",
-                "from wood or metal. Catching vermin requires trap to be set with bait."},
+                "of wood or metal. Catching vermin requires trap to be set with bait."},
```

Comment:
"built of" just doesn't sound right to me. It's not the building material in its original form any more. Maybe "made of" would be better.